src/WinGet.RestSource/Models/Schemas/Information.cs

```diff
@@ -21,21 +21,18 @@ public class Information
         /// </summary>
         public Information()
         {
-            this.SourceIdentifier = string.IsNullOrEmpty(ApiConstants.SourceIdentifier) ? "DefaultSourceIdentifier" : ApiConstants.SourceIdentifier;
+            if (string.IsNullOrEmpty(ApiConstants.SourceIdentifier))
+            {
+                throw new System.ArgumentNullException("SourceIdentifier is not configured.");

```

Comment:
IIRC, the source identifier maps to the server identifier environment variable? Lets make the error message helpful in guiding folks to to the specific thing that needs to be updated/fixed to work with this setup.